src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackSubmitPageUiTest.java

```diff
@@ -367,181 +364,213 @@ private void testQuestionTypesSubmitAction(){
+        assertEquals("10 points left to distribute. The same amount of points should not be given multiple times.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 0, "26");
-        assertEquals("9 points left to distribute.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("9 points left to distribute.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 0, "50");
-        assertEquals("Over allocated 15 points.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("Over allocated 15 points.", submitPage.getConstSumMessage(qnNumber, 0));
         submitPage.fillResponseTextBox(qnNumber, 0, 1, "50");
-        assertEquals("Over allocated 25 points. The same amount of points should not be given multiple times.",submitPage.getConstSumMessage(qnNumber, 0));
+        assertEquals("Over allocated 25 points. The same amount of points should not be given multiple times.", submitPage.getConstSumMessage(qnNumber, 0));
         
-        //For other const sum question, just test one message.
+        // For other const sum question, just test one message.
         qnNumber = 18;
-        assertEquals("100 points left to distribute.",submitPage.getConstSumMessage(qnNumber, 3));
-        
+        assertEquals("100 points left to distribute.", submitPage.getConstSumMessage(qnNumber, 3));
     }
     
-    private void testContribSubmitAction(){
+    private void testContribSubmitAction() {
         ______TS("test submit actions for contribution questions.");
         
-        //No tests from instructor since contribution questions are only from students to own team members.
-        //Test by logging in as student instead.
-        
+        // No tests from instructor since contribution questions are only from students to own team members.
+        // Test by logging in as student instead.
         
-        //Test input disabled
+        // Test input disabled
         submitPage = loginToStudentFeedbackSubmitPage("Danny", "Closed Session");
+        
         int qnNumber = 1;
         int responseNumber = 0;
+        
         assertTrue(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_TEXT + "-" + qnNumber + "-" + responseNumber));
         
+        // Test that the recipient selection is disabled and not visible
+        assertFalse(submitPage.isNamedElementVisible(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
+        assertFalse(submitPage.isNamedElementEnabled(Const.ParamsNames.FEEDBACK_RESPONSE_RECIPIENT + "-" + qnNumber + "-" + responseNumber));
     }
     
-    private void fillResponseTextBoxWithRecheck(int qnNumber, int responseNumber, String text, String expected) {
```

Comment:
this is the method, the counter is never looped over, I was looking at teammates from a year ago, it was used somewhere but it is no longer used where it was originally used, it was then used at the other place where I'm going to comment to point reviewers to but I don't think that was the purpose of this method.